src/cuda_packet.cpp

```diff
@@ -121,13 +124,117 @@ void jitc_cuda_render_gather_packet(const Variable *v, const Variable *ptr,
     }
 }
 
+/**
+ * Render the code required to scatter reduce a packet of variables.
+ */
+void jitc_cuda_render_scatter_reduce_packet(const Variable *v,
+                                            const Variable *ptr,
+                                            const Variable *index,
+                                            const Variable *mask) {
+    bool is_masked         = !mask->is_literal() || mask->literal != 1;
+    PacketScatterData *psd = (PacketScatterData *) v->data;
+    const std::vector<uint32_t> &values = psd->values;
+    const Variable *v0                  = jitc_var(values[0]);
+
+    const ThreadState *ts = thread_state_cuda;
+
+    uint32_t count = (uint32_t) values.size(),
+             tsize = type_size[v0->type];
+
+    if (count % 2 != 0)
+        jitc_fail("jitc_cuda_render_scatter_reduce_packet(): Number of "
+                  "elements not supported for reduction.");
+
+    if (ts->compute_capability >= 90) {
+        // Use the new `red.global.v2` instructions. This enables both min & max
```

Comment:
Is this comment up-to-date? `.v2` is only two elements, and we're doing wider ones AFAIK.